mobile-widgets/qmlmanager.cpp

```diff
@@ -1436,14 +1434,9 @@ QStringList QMLManager::buddyInit() const
 	int i = 0;
 	for_each_dive (i, d) {
 		QString temp = d->buddy;
-		if (!temp.isEmpty() && !temp.contains(",")){
+		if (!temp.isEmpty()){
 			buddies << d->buddy;
 		}
-		else if (!temp.isEmpty()){
-			QRegExp sep("(,\\s)");
-			QStringList tempList = temp.split(sep);
-			buddies << tempList;
-		}
```

Comment:
Because this is what you actually want, you want buddies to hold a list of the buddies